tests/c/return_char.c

```diff
@@ -1,50 +0,0 @@
-// Run-time:
```

Comment:
Is it worth keeping some of these tests around anyway, even if they don't test an SGI?

Reply:
In general I think "more tests" is better than "fewer tests" with one proviso: there's not much use having a load of tests that are really the same test in disguise (i.e. they all test the same thing) because it gives a misleading impression of coverage. I don't know if that's the case here: I'll be happy with whatever @ptersilie decides.

Reply:
I've removed these because they don't really test anything that other tests don't already cover. Just look at the loop the interpreter is in. All it does is increment `i`. There's plenty of tests that already do that.